src/borg/archiver.py

```diff
@@ -1616,7 +1616,8 @@ def do_compact(self, args, repository):
         # see the comment in do_with_lock about why we do it like this:
         data = repository.get(Manifest.MANIFEST_ID)
         repository.put(Manifest.MANIFEST_ID, data)
-        repository.commit(compact=True, cleanup_commits=args.cleanup_commits)
+        percent = args.threshold / 100
```

Comment:
percent is a misnomer / misleading.

guess args.threshold is a percent value (e.g. 5 would mean 5%).
but if you divide that by 100, it is not a percent value any more.

so, just `threshold`?